### PR TITLE
[1.2.x] Scala 2.13.0-M5

### PR DESCRIPTION
**Pull request by eed3si9n**
This is a backport of https://github.com/sbt/io/pull/199
Also a continuation of https://github.com/sbt/io/pull/205

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=1.2.x&zinc=1.2.x&io=pull/206/head&librarymanagement=1.2.x&util=1.2.x&website=1.2.x&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

**Comment by typesafe-tools** [comment]
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1213/console">is in progress...</a>

File: project/Dependencies.scala [hunk 1]
```diff
@@ -4,18 +4,13 @@ import Keys._
...
   val scalaCompiler = Def.setting { "org.scala-lang" % "scala-compiler" % scalaVersion.value }
 
   val scalaCheck = "org.scalacheck" %% "scalacheck" % "1.14.0"
-  val scalatest = Def.setting {
-    if (scalaVersion.value == "2.13.0-M4")
-      "org.scalatest" %% "scalatest" % "3.0.6-SNAP1"
-    else
-      "org.scalatest" %% "scalatest" % "3.0.5-M1"
-  }
+  val scalatest = "org.scalatest" %% "scalatest" % "3.0.6-SNAP3"
```
Comment (cunei) [inline project/Dependencies.scala]: You need SNAP5, here

Comment (eed3si9n) [reply project/Dependencies.scala]: Good catch!

**Comment by typesafe-tools** [comment]
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1215/console">is in progress...</a>

**Comment by typesafe-tools** [comment]
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1216/console">is in progress...</a>

**Comment by typesafe-tools** [comment]
[//]: # (Refs: 1.2.x-1.2.x-pull/206/head-1.2.x-1.2.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1215/">has checked</A> the following projects, tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/tree/1.2.x">1.2.x               </A> | sbt/sbt@dedc7a53e6c803fd54c1ce5c8f6f8ad00a0de342
  zinc              | <A HREF="https://github.com/sbt/zinc/tree/1.2.x">1.2.x               </A> | sbt/zinc@872952ac6483db259ea13b3198c8047c541aa4ea
  io                | <A HREF="https://github.com/sbt/io/pull/206">pull/206/head       </A> | sbt/io@9641a9fd25a826fc6a6a2295aa55453169c0a768
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/tree/1.2.x">1.2.x               </A> | sbt/librarymanagement@a56242036e1c3463bcf90744fa8cb392fdce6d09
  util              | <A HREF="https://github.com/sbt/util/tree/1.2.x">1.2.x               </A> | sbt/util@016858ddd8601f9d55224d2b6ec9b046cf8e29d9

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=1.2.x&zinc=1.2.x&io=pull/206/head&librarymanagement=1.2.x&util=1.2.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

**Comment by eed3si9n** [comment]
The failure was in lm's unit testing, so I am guessing it's unrelated.

**Pull request merged**